react/src/components/blocks/LanguageSwitcher/LanguageSwitcher.js

```diff
@@ -1,11 +1,28 @@
 import React from 'react'
+
+import Link from '@/ui/Link'
+
+// styles
 import styles from './LanguageSwitcher.module.css'
 
 function LanguageSwitcher() {
   return (
-    <>
-      <p className={styles.line}>LanguageSwitcher goes here.</p>
-    </>
+    <div className={styles.container}>
+      <div className={styles.langListContainer}>
+        <div className={styles.langList}>
+          Google offered in:&nbsp;
+          <Link className={styles.langLink} title="हिन्दी" />
```

Comment:
put this value outside component and iterate with map

Reply:
resolved comment. kindly check @ramsunvtech.

Reply:
Done @ramsunvtech. Kindly check.